src/doc/book/primitive-types.md

```diff
@@ -171,12 +171,10 @@ and a length.
 ## Slicing syntax
 
 You can use a combo of `&` and `[]` to create a slice from various things. The
-`&` indicates that slices are similar to [references], which we will cover in
+`&` indicates that slices are similar to [references][references], which we will cover in
```

Comment:
But the `[references]` style cannot be convert to PDF/LaTex correctly with Pandoc. And the converted result is still `[references]` in the LaTex source instead of `references` with a link reference.

If it's `[references][references]`  or `references][]` as defined in the [Markdown Syntax][markdown-link], the Pandoc tools can convert it to LaTex or PDF correctly.

[markdown-link]: https://daringfireball.net/projects/markdown/syntax#link